### PR TITLE
Update Patched VS Code version to 1.90.1 with patches applied

### PR DESCRIPTION
**Pull request by navinsoni**
*Description of changes:*
This PR adds patches for code editor and applies them on Patched VS Code.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -27,10 +32,23 @@ printf "\n======== Applying patches ========\n"
...
+cp ../LICENSE-THIRD-PARTY .
+cd ${PROJ_ROOT}
+
 # Comment out breaking lines in postinstall.js
 printf "\n======== Comment out breaking git config lines in postinstall.js ========\n"
 sh ${PROJ_ROOT}/scripts/postinstall.sh
 
+# Build tarball for conda feedstock from vscode dir
+printf "\n======== Build Tarball for Conda Feedstock ========\n"
+bash ${PROJ_ROOT}/scripts/create_code_editor_tarball.sh -v 1.2.0
```
Comment (aws-pangestu) [inline scripts/install.sh:50]: Can we make the version number an input instead of hard coded

Comment (navinsoni) [reply scripts/install.sh:50]: yup I was thinking of the same. 
I was thinking we should have a version file and use that to take version value at all places

File: scripts/install.sh [hunk 2]
```diff
@@ -10,6 +10,10 @@ PROJ_ROOT=$(pwd)
 printf "\n======== Cleaning out patches ========\n"
 quilt pop -a
 
+# empty vscode module
+printf "\n======== Delete data is vs code module if present ========\n"
```
Comment (aws-asolidu) [inline scripts/install.sh:15]: typo? "if" instead of "is"

File: scripts/install.sh [hunk 3]
```diff
@@ -40,6 +48,6 @@ printf "\n======== Deleting vscode/node_modules ========\n"
 rm -rf "${PROJ_ROOT}/vscode/node_modules"
 
 # Build the project
-printf "\n======== Building project in ${PROJ_ROOT}/vscode ========\n"
+#printf "\n======== Building project in ${PROJ_ROOT}/vscode ========\n"
```
Comment (aws-asolidu) [inline scripts/install.sh]: Why is this commented out?

File: scripts/create_code_editor_tarball.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+if [[ -z $version ]]; then
+ echo "Please provide version using '-v'";
+ exit 1
+fi
+
+VERSION=$version
+# Set current project root
+PROJ_ROOT=$(pwd)
+
+mkdir -p ${PROJ_ROOT}/sagemaker-code-editor/code-editor${VERSION}
```
Comment (aws-asolidu) [inline scripts/create_code_editor_tarball.sh:28]: could we create a local variable for this path since we use it in many places?

**Pull request merged**